### PR TITLE
Initial YAML Build

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"
+
+jobs:
```
Comment (jack-education) [inline azure-pipelines.yml:21]: We can have a single job on the DAS - Continuous Integration pool, with a demand to be Windows like https://github.com/SkillsFundingAgency/das-alpha-spikes/blob/master/FATv2/APIM/SFA.DAS.FATv2.Alpha/azure-pipelines.yml#L20-L22 - a recent change

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- job: 'CodeBuild'
+  pool:
+    name: 'DAS - Continuous Integration'
+  workspace:
+    clean: all
+  steps:
+  - task: gittools.gitversion-preview.gitversion-task.GitVersion@5
```
Comment (jack-education) [inline azure-pipelines.yml:28]: There's a new UseGitVersion task that we can use as GitVersion is being deprecated https://github.com/SkillsFundingAgency/das-alpha-spikes/blob/master/FATv2/APIM/SFA.DAS.FATv2.Alpha/azure-pipelines.yml#L25

File: azure-pipelines.yml (hunk 1 above)
Comment (mwevill) [reply azure-pipelines.yml]: Refactored to use this,

File: azure-pipelines.yml (hunk 2 above)
Comment (mwevill) [reply azure-pipelines.yml]: Refactored

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+- task: DotNetCoreCLI@2
+  displayName: 'Publish API'
+  inputs:
+    command: publish
+    publishWebProjects: false
+    projects: 'src/SFA.DAS.ContentApi.Api/SFA.DAS.ContentApi.Api.csproj'
+    arguments: '--configuration $(buildConfiguration) --output $(build.artifactstagingdirectory)/publish --no-restore --no-build'
+
+- task: CopyFiles@2
```
Comment (jack-education) [inline azure-pipelines.yml:66]: Could this task be moved and combined with the below CopyFiles@2 task?

Comment (mwevill) [reply azure-pipelines.yml]: Sorted (And removed duplicate publish Artifact)

**Pull request merged**